src/nix.rs

```diff
@@ -296,7 +295,9 @@ impl CallOpts {
 
         if output.status.success() {
             let stdout: &[u8] = &output.stdout;
-            let paths: Vec<PathBuf> = ::nix::parse_nix_output(stdout, PathBuf::from)?;
+            let paths: Vec<PathBuf> = osstrlines::Lines::from(stdout)
+                .map(|line| line.map(PathBuf::from))
+                .collect::<Result<Vec<PathBuf>, _>>()?;
```

Comment:
This is a lot better, putting the function was very much fighting the system. I wasn’t sure how much work an iterator would be, but it seems to be totally reasonable.